steps/generate_target_distribution_test.py

```diff
@@ -0,0 +1,57 @@
+import os
```

Comment:
I very much appreciate these tests, but we are moving away from testing step functions, so this file can be removed. Apologies for the confusion